### PR TITLE
[StateService]check magic in console command

### PR DESCRIPTION
**Pull request by ZhangTao1596**
Currently we don't initialize state service,  if `Network` in state service is not equal `Magic`.
So that we should disable the console commands too.

### CODE REVIEW

**Review by shargon — DISMISSED** [review]

File: src/StateService/StatePlugin.cs [hunk 1]
```diff
@@ -80,6 +80,7 @@ void IPersistencePlugin.OnPersist(NeoSystem system, Block block, DataCache snaps
         [ConsoleCommand("start states", Category = "StateService", Description = "Start as a state verifier if wallet is open")]
         private void OnStartVerifyingState()
         {
+            if (System.Settings.Magic != Settings.Default.Network) throw new InvalidOperationException("Network doesn't match");
```
Comment (cloud8little) [inline src/StateService/StatePlugin.cs:83]:
```suggestion
            if (system.Settings.Magic != Settings.Default.Network) throw new InvalidOperationException("Network doesn't match");
```

Comment (ZhangTao1596) [reply src/StateService/StatePlugin.cs]: There is no field named `system`;
I add `System` check.

**Comment by cloud8little** [comment]
Test passed.

592862cd7c2e3f2da12940b5ef6b410

**Review by shargon — APPROVED** [review]

**Review by cloud8little — APPROVED** [review]

**Comment by superboyiii** [comment]
@erikzhang Could you merge it? Seems work flow is stuck.

**Pull request merged**